src/components/flat-table/flat-table.style.js

```diff
@@ -95,6 +95,15 @@ StyledFlatTableRoot.defaultProps = {
 };
 
 const StyledFlatTableWrapper = styled.div`
+  ${({ hasStickyHead }) =>
```

Comment:
There's already a function like this for `hasStickyHead` in `StyledFlatTableWrapper` - you should just put this CSS in there instead 👍🏻

Reply:
@sjohnson-sage good point! :)